api/types/server_test.go

```diff
@@ -279,7 +307,7 @@ func TestServerCheckAndSetDefaults(t *testing.T) {
 				},
 			},
 			assertion: func(t *testing.T, s *ServerV2, err error) {
-				require.EqualError(t, err, `Addr must be set when server SubKind is "openssh"`)
+				require.EqualError(t, err, `addr must be set when server SubKind is "openssh"`)
```

Comment:
Might I suggest `require.ErrorContains`, so the test is less brittle?

```suggestion
				require.ErrorContains(t, err, "addr must be set")
```

(Or maybe even just "addr" would be enough.)

Same for others.